src/Examples/PiTop.Interactive.Rover/RoverRobotSetup.cs

```diff
@@ -65,36 +65,51 @@ private static async Task ConfigureRover(CSharpKernel csharpKernel)
             await csharpKernel.SetVariableAsync(nameof(resourceScanner), resourceScanner);
 
             var source = new CancellationTokenSource();
+            await csharpKernel.SetVariableAsync(nameof(source), source);
 
             var robotLoop = Task.Run(() =>
             {
+                Console.WriteLine("starting robotLoop");
```

Comment:
don't use Console.WriteLine for logging ,this is captured and flows on the output

Reply:
We should use the [PR](https://github.com/pi-top/pi-top-4-.NET-Core-API/pull/32)